src/wasm-lib/kcl/src/unparser.rs

```diff
@@ -334,8 +334,25 @@ impl CallExpressionKw {
                 .iter()
                 .map(|arg| arg.recast(options, indentation_level, ctxt)),
         );
-        let args = arg_list.join(", ");
-        format!("{indent}{name}({args})")
+        let args = arg_list.clone().join(", ");
+        if arg_list.len() >= 4 || (arg_list.len() >= 2 && args.len() >= 60) {
```

Comment:
Probably post-1.0, but it would be nice if we implemented real length-aware pretty printing.